src/components/v1/V1Project/ProjectActivity/eventElems/TapEventElem.tsx

```diff
@@ -57,6 +60,13 @@ export default function TapEventElem({
 
   if (!event) return null
 
+  const handleProject = (projectId: number, allocator: string) => {
+    if (allocator.toLowerCase() === V1_V3_ALLOCATOR_ADDRESS.toLowerCase()) {
+      return <V2V3ProjectHandleLink projectId={projectId} />
```

Comment:
**question** This `V2V3ProjectHandleLink` component relies on `V2V3ContractContext`, which I don't think we provide for V1 projects. Not sure how this can work (but I may be missing something!)

Reply:
Alternatively, we could just render the Project ID and append the V3 version badge:
```
Project #69 <V3>
```

WDYT?

Reply:
It is working, I was trying to find where we pass provider but couldn't find it, though we are using the same component in `Mod` component in the v1 folder. should I do the same there or keep it as it is now? 
I can add a badge in any of these cases.

Reply:
@NakedFool i don't think it's quite working as expected.

V2V3ProjectHandleLink uses `useProjectHandleText`, which uses `useProjectHandle`, which relies on V2V3ContractsContext. If I log `console.log(useContext(V2V3ContractsContext))` in `useProjectHandle`, we can see it is undefined (we're not providing it).

Therefore `useProjectHandle` will never actually return a handle. We can verify this by looking at the Peel V1 project. Our Peel V3 project has a handle `@peel`, so any payout to that project should render `@peel`. But the latest activity item has a distribution to our V3 project, and doesn't render our handle:

<img width="504" alt="Screenshot 2023-01-25 at 11 54 15 AM" src="https://user-images.githubusercontent.com/12551741/214463066-3e73ff70-dd37-4dd0-9f15-b4f10c53f001.png">

FWIW i think is probably okay, but I don't think we should be using the `V2V3ProjectHandleLink` component here. I think
lets render the Project ID and append the V3 version badge in a new component. Something like `V2V3ProjectLink`.

WDYT about this?

<img width="482" alt="Screenshot 2023-01-25 at 12 01 26 PM" src="https://user-images.githubusercontent.com/12551741/214464011-5e7c5aac-ee11-4d78-9a67-6f1bea171c62.png">

Reply:
Oh oh oh, gotcha! 
Basically what you are saying is why to use `V2V3ProjectHandleLink` if we can never get the handle back for now.

Thanks very much for the thorough explanation, I've created a light component `V2V3ProjectLink` that requires `projectId` but `allocator` is optional, that way we can decide if we want to include the version badge or not.

I've also added it to `Mod` component to keep things consistent.